content/applications/hr/payroll/payslips.rst

```diff
@@ -0,0 +1,299 @@
+:guilabel:`Cancel Payslip` smart button. Click the :guilabel:`Edit` smart button to make
+modifications, and click the :guilabel:`Save` smart button to save the information after edits have
+been made.
+
+Once everything is correct, click the :guilabel:`Create Draft Entry` smart button to create the
+payslip. The chatter will be updated to show the email automatically sent to the employee, along
+with the PDF of the payslip.
+
+.. image:: payslips/payslip-chatter.png
+   :align: center
+   :alt: The new payslip is emailed to the employee and the email appears in the chatter.
+
+Next, payment must be sent to the employee. Click the :guilabel:`Make Payment` smart button, and a
+pop up will appear asking which *Bank Journal* the payment should be made against. Select the bank
+journal from the drop down, then click the :guilabel:`Confirm` smart button.
+
+.. image:: payslips/make-payment.png
+   :align: center
+   :alt: Click Make Payment to send the payment to the employee.
+
+If a payment needs to be cancelled or refunded, click the corresponding :guilabel:`Refund` or
+:guilabel:`Cancel Payslip` smart buttons.
+
+All payslips
+============
+
+To view all the payslips regardless of status, go to :menuselection:`Payroll --> Payslips -->
+All Payslips`. In this view, all payslips are organized by batch (in a default list view).
+
+Click on the arrow next to the individual batch name to view all the payslips in that particular
+batch, along with all the payslip details. The number of payslips in the batch is written in
+parenthesis after the batch name. The status for each individual payslip will appear on the far
+right, indicating if it is in draft mode or if it is done.
+
+.. image:: payslips/all-payslips.png
+   :align: center
+   :alt: View all payslips organized by batches. Click on the arrow to expand each batch.
+
+Click on an individual payslip to view the details for that payslip. Using the breadcrumb menu,
+click *Employee Payslips* to go back to the list view of all payslips.
+
+A new payslip can be created by clicking the :guilabel:`Create` smart button, and enter all the
+information as described in the :ref:`Create a new payslip <payroll/new-payslip>` section.
+
+To print payslips to a PDF, first select the payslips by clicking on the individual checkbox next to
+each payslip to print, or clicking on the box next to :guilabel:`Reference`, which will select all
+visible payslips. Click on the :guilabel:`Print` smart button to print the payslips to a PDF file.
+
+.. image:: payslips/print.png
+   :align: center
+   :alt: Click on the Print smart button to print payslips to a PDF.
+
+Payslips can also be exported to an Excel spreadsheet. When exporting, all paylsips will be exported
+regardless if some are checked off/sletected or not. Click on the :guilabel:`Export All` smart
+button (download icon) to export all payslips to an Excel spreadsheet.
```

Comment:
I looked, too! Nothing quite like it, and since it is a bit specific and not universal, like the trash can I used (it was a stretch since our can is different from the emoji on) I figured best not to put something in that didn't fit.